src/lwc/geFormService/geFormService.js

```diff
@@ -111,6 +115,30 @@ class GeFormService {
         return this.objectMappings[objectDevName];
     }
 
+    /**
+     * Gets the Custom Label that was used on the form template for the Donation Amount field
+     * @return {string}
+     */
+    getDonationAmountCustomLabel() {
+        // find field that is mapped to Opportunity Amount
+        const mapping = this.getFieldMappingWrapperFromTarget(`${OPPORTUNITY_OBJECT.objectApiName}.${OPPORTUNITY_AMOUNT.fieldApiName}`);
+        const mappingDevName = mapping.DeveloperName;
+        // get developer name of mapping cmt
+        let fieldElement;
+        for(const section of this.formTemplate.layout.sections) {
```

Comment:
Just a note that I've found myself doing more and more of this section and fields looping also. Feels like a spot we could create a utility method for in the future.

Reply:
A way to flatten a full layout down to an array of elements, maybe? Or a class that accepts a form layout as a constructor and has methods that allow you to retrieve sections, elements etc? I agree but not sure which fits better.